pom.xml

```diff
@@ -7,7 +7,7 @@
 
     <groupId>io.github.pmckeown</groupId>
     <artifactId>dependency-track-maven-plugin</artifactId>
-    <version>0.8.1</version>
```

Comment:
If you target the develop branch this change is not necessary

Reply:
I've created another PR and reverted the SNAPSHOT change